src/yoroi-wallets/types/tokens.ts

```diff
@@ -86,3 +86,35 @@ export type TokenRegistryEntry = {
   logo?: string
   decimals?: number
 }
+
+export type NFTMetadata = {
+  name: string
+  image: string | Array<string>
+  mediaType?: string
+  description?: string | Array<string>
+  authors?: string
+  author?: string
+  files?: Array<{
+    name?: string
+    mediaType?: string
+    src?: string | Array<string>
+  }>
+}
+
+export type CardanoAssetMintMetadata = {
+  // transaction_metadatum_label: 721 for NFTs
+  // See CIP 721
+  // https://github.com/cardano-foundation/CIPs/blob/8b1f2f0900d81d6233e9805442c2b42aa1779d2d/CIP-NFTMetadataStandard.md
+  [key: string]: {
```

Comment:
what is key?

Reply:
`export type CardanoAssetMintMetadata = { [key: string]: CardanoAssetMetadata }`?